README.md

```diff
@@ -3,9 +3,9 @@
 ## How to build and run
 
 ```
-$ cd $GOPATH
-$ go get github.com/awalterschulze/gographviz
-$ cd ./src/github.com/davrodpin/ovsdbviz # you have to clone or go get it
+$ cd $GOPATH/src
+$ git clone https://github.com/davrodpin/ovsdbviz.git
+$ cd ovsdbviz
```

Comment:
The ovsdbviz source directory inside GOPATH should follow the whole github repo address:

```
$ cd $GOPATH/src
$ mkdir -p github.com/davrodpin && cd github.com/davrodpin
$ git clone https://github.com/davrodpin/ovsdbviz.git
$ cd ovsdbviz
```